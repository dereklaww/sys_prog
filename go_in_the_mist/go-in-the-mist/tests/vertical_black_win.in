who
place A1
place B1
place A2
place B2
place A3
place B3
view
place A4
place B4
history
who
place A5