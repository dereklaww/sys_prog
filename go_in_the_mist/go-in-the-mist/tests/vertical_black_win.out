B
L1,............................xxxxxxxxxxxxxxxxxxxxx
A1B1A2B2A3B3A4B4
B
Black wins!
A1B1A2B2A3B3A4B4A5